OverlaysPatches/BlindOverlayPatch.cs

```diff
@@ -0,0 +1,18 @@
+using HarmonyLib;
+using System.Reflection;
+
+namespace Overlays {
+    [HarmonyPatch]
+    public static class BlindOverlayPatch // Отключаем эффект от ослепления, выключая метод Draw https://github.com/space-wizards/space-station-14/blob/master/Content.Client/Eye/Blinding/BlindOverlay.cs
+    {
+        private static MethodBase TargetMethod() {
+            var BlindOverlay = MarseyPatch.ContentClient.GetType("Content.Client.Eye.Blinding.BlindOverlay")!;
+            return BlindOverlay.GetMethod("Draw", BindingFlags.NonPublic | BindingFlags.Instance);
+        }
```

Comment:
```suggestion
            Type? BlindOverlay = MarseyPatch.ContentClient.GetType("Content.Client.Eye.Blinding.BlindOverlay");
            if (BlindOverlay != null)
            {
            	MethodInfo? BlindOverlayDraw = BlindOverlay.GetMethod("Draw", BindingFlags.NonPublic | BindingFlags.Instance);
            	if (BlindOverlayDraw != null)
        		yield return BlindOverlayDraw;
            }
        }
```

You should check if that type/method exists, else it will crash by trying to patch a non-existent method.